core/widgetdiv.js

```diff
@@ -147,6 +146,5 @@ Blockly.WidgetDiv.position = function(anchorX, anchorY, windowSize,
   }
   Blockly.WidgetDiv.DIV.style.left = anchorX + 'px';
   Blockly.WidgetDiv.DIV.style.top = anchorY + 'px';
-  Blockly.WidgetDiv.DIV.style.height =
-      (windowSize.height - anchorY + scrollOffset.y) + 'px';
+  Blockly.WidgetDiv.DIV.style.height = windowSize.height + 'px';
```

Comment:
As the widget div is now inserted relative to injection div, this is now
quite more simple to manage. I've tested with multiple and scrolled
workspaces


For the menu height, I don't remember why. I'll check for it, maybe it is
an issue due to my github misunderstanding. (merging, syncing, pulling are
new for me and I may have introduce some error)



2016-01-21 9:48 GMT+01:00 Carlos <notifications@github.com>:

> In core/widgetdiv.js
> <https://github.com/google/blockly/pull/181#discussion_r50373179>:
>
> > @@ -147,6 +146,5 @@ Blockly.WidgetDiv.position = function(anchorX, anchorY, windowSize,
> >    }
> >    Blockly.WidgetDiv.DIV.style.left = anchorX + 'px';
> >    Blockly.WidgetDiv.DIV.style.top = anchorY + 'px';
> > -  Blockly.WidgetDiv.DIV.style.height =
> > -      (windowSize.height - anchorY + scrollOffset.y) + 'px';
> > +  Blockly.WidgetDiv.DIV.style.height = windowSize.height + 'px';
>
> Reverting back to the old widget height will cause issues with multiple or
> scrolled workspaces.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/blockly/pull/181/files#r50373179>.
>